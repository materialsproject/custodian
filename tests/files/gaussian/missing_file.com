%chk=DME.chk
%mem=48GB
%NProcShared=28
#P PBE1PBE/6-31+G* opt scf=(maxcycle=100) geom=check guess=read

H10 C4 O2

0 1



