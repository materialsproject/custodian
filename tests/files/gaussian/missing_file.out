 Entering Gaussian System, Link 0=g16
 Initial command:
 /gpfs/software/gaussian/g16/l1.exe "/gpfs/scratch/ratwi/tests/Custodian/missing_file/new/Gau-27979.inp" -scrdir="/gpfs/scratch/ratwi/tests/Custodian/missing_file/new/"
 Entering Link 1 = /gpfs/software/gaussian/g16/l1.exe PID=     27981.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                25-Mar-2024 
 ******************************************
 %chk=DME.chk
 %mem=48GB
 %NProcShared=28
 Will use up to   28 processors via shared memory.
 ---------------------------------------------------------------
 #P PBE1PBE/6-31+G* opt scf=(maxcycle=100) geom=check guess=read
 ---------------------------------------------------------------
 1/18=20,19=15,26=3,29=2,38=1/1,3;
 2/9=110,12=2,40=1/2;
 3/5=1,6=6,7=11,11=2,14=-4,25=1,30=1,71=1,74=-13,116=-2/1,2,3;
 4/5=1/1;
 5/5=2,7=100,38=6/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 7//1,2,3,16;
 1/18=20,19=15,26=3/3(2);
 2/9=110/2;
 99//99;
 2/9=110/2;
 3/5=1,6=6,7=11,11=2,14=-2,25=1,30=1,71=1,74=-13/1,2,3;
 4/5=5,16=3,69=1/1;
 5/5=2,7=100,38=5/2;
 7//1,2,3,16;
 1/18=20,19=15,26=3/3(-5);
 2/9=110/2;
 6/7=2,8=2,9=2,10=2,19=2,28=1/1;
 99/9=1/99;
 Leave Link    1 at Mon Mar 25 02:14:19 2024, MaxMem=  6442450944 cpu:               3.9 elap:               0.2
 (Enter /gpfs/software/gaussian/g16/l101.exe)
 ---------
 H10 C4 O2
 ---------
 Structure from the checkpoint file:  "DME.chk"
 FileIO operation on non-existent file.
 FileIO: IOper= 2 IFilNo(1)=-20997 Len=          20 IPos=           0 Q=  140737488312672


 dumping /fiocom/, unit = 1 NFiles =    15 SizExt =    524288 WInBlk =   1048576
                   defal = F LstWrd =    16777216 FType=2 FMxFil=10000

 Number            501           502           598           801           988           989
 Base          7340032      12582912      13631488      15728640       9437184       8388608
 End           7341032      12587017      13631490      15728646       9507188       8408608
 End1          8388608      13631488      14680064      16777216      10485760       9437184
 Wr Pntr       7340032      12583016      13631488      15728640       9437184       8388608
 Rd Pntr       7340032      12582912      13631488      15728640       9507188       8388608
 Length           1000          4105             2             6         70004         20000
 Int.Num             1             2             3             4             5             6

 Number            991           992           993           994           995           996
 Base         11534336      10485760       6291456       1048576       5242880       3145728
 End          11540898      10485770       6291656       1048606       5242900       3145928
 End1         12582912      11534336       7340032       2097152       6291456       4194304
 Wr Pntr      11534336      10485760       6291456       1048576       5242880       3145728
 Rd Pntr      11540898      10485770       6291656       1048606       5242900       3145928
 Length           6562            10           200            30            20           200
 Int.Num             7             8             9            10            11            12

 Number            997           998           999
 Base          4194304       2097152      14680064
 End           4194474       2097352      14782568
 End1          5242880       3145728      15728640
 Wr Pntr       4194304       2097352      14680064
 Rd Pntr       4194474       2097352      14682568
 Length            170           200        102504
 Int.Num            13            14            15


 dumping /fiocom/, unit = 2 NFiles =     1 SizExt =         0 WInBlk =   1048576
                   defal = F LstWrd =     2097152 FType=2 FMxFil=10000

 Number              0
 Base          1048576
 End           2097152
 End1          2097152
 Wr Pntr       1048576
 Rd Pntr       1048576
 Length        1048576
 Int.Num             1


 dumping /fiocom/, unit = 3 NFiles =     1 SizExt =    524288 WInBlk =   1048576
                   defal = F LstWrd =     2097152 FType=2 FMxFil=10000

 Number              0
 Base          1048576
 End           2097152
 End1          2097152
 Wr Pntr       1048576
 Rd Pntr       1048576
 Length        1048576
 Int.Num             1
 FileIO operation on non-existent file.
 FileIO: IOper= 2 IFilNo(1)=-20997 Len=          20 IPos=           0 Q=  140737488312672
 Error termination in NtrErr:
 NtrErr Called from FileIO.
